source/agora/test/Base.d

```diff
@@ -969,14 +962,13 @@ public class TestAPIManager
 
     ***************************************************************************/
 
-    void addBlock (Height enrolled_height = Height(0),
-        string file = __FILE__, size_t line = __LINE__)
+    void addBlock (string file = __FILE__, size_t line = __LINE__)
```

Comment:
The `enrolled_height` remains in the doc, it's removed though.

Reply:
Updated